bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/TcsEphemerisExport.scala

```diff
@@ -169,11 +168,19 @@ object TcsEphemerisExport {
           p  <- files.write(hid, em)
         } yield p
 
-      def lookupEphemeris(hid: HorizonsDesignation): TryExport[EphemerisMap] =
-        HorizonsService2.lookupEphemerisE[EphemerisElement](hid, site, new Date(start.toEpochMilli), new Date(end.toEpochMilli), ElementCount) {
+      def lookupEphemeris(hid: HorizonsDesignation): TryExport[EphemerisMap] = {
+        val s = start.toEpochMilli
+        val e = end.toEpochMilli
+        HorizonsService2.lookupEphemerisE[EphemerisElement](hid, site, new Date(s), new Date(e), ElementCount) {
```

Comment:
Yeah we have a lot of stuff that goes back and forth between `Long` and `Date` because of old APIs. Would be nice to fix.